kubemarine/core/flow.py

```diff
@@ -146,7 +144,7 @@ def run_tasks(resources: res.DynamicResources, tasks, cumulative_points=None, ta
         return
 
     init_tasks_flow(cluster)
-    run_flow(filtered_tasks, cluster, cumulative_points)
+    run_flow(tasks, final_list, cluster, cumulative_points, [])
```

Comment:
I suggest to always run scheduled cumulative points at the right moment. To see the advantages, refer to `Test Case 2` from the description.

It imply that KubeMarine runs will be slower in some cases, especially if the task is run over the existing cluster and graceful reboot is required.

On the other hand, only scheduled reboot is affected, and it should be scheduled only when necessary and only when the changes in the system are detected. See another part of the PR.

Reply:
ok. @OleksandrIaresko  pls check the performance impact